test/remote/account_signin_verification_tests.js

```diff
@@ -178,172 +178,6 @@ TestServer.start(config)
     )
 
     test(
-      'Unverified account becomes verified from sign-in verification',
-      function (t) {
-        var email = server.uniqueEmail()
-        var password = 'allyourbasearebelongtous'
-        var client = null
-        var uid
-        var code
-        return Client.create(config.publicUrl, email, password)
-          .then(
-            function (x) {
-              client = x
-              t.ok(client.authAt, 'authAt was set')
-            }
-          )
-          .then(
-            function () {
-              return client.emailStatus()
-            }
-          )
-          .then(
-            function (status) {
-              t.equal(status.verified, false, 'new account is unverified')
-              t.equal(client.emailVerified, false, 'new account email unverified')
-            }
-          )
-          .then(
-            function () {
-              return server.mailbox.waitForEmail(email)
-            }
-          )
-          .then(
-            function (emailData) {
-              t.equal(emailData.subject, 'Verify your Firefox Account')
-            }
-          )
-          .then(
-            function () {
-              return client.login({keys:true})
-            }
-          )
-          .then(
-            function () {
-              return server.mailbox.waitForEmail(email)
-            }
-          )
-          .then(
-            function (emailData) {
-              uid = emailData.headers['x-uid']
-              code = emailData.headers['x-verify-code']
-              t.equal(emailData.subject, 'Confirm new sign-in to Firefox')
-              t.ok(uid, 'sent uid')
-              t.ok(code, 'sent verify code')
-            }
-          )
-          .then(
-            function () {
-              return client.emailStatus()
-            }
-          )
-          .then(
-            function (status) {
-              t.equal(status.verified, false, 'account is not verified, unverified sign-in')
-            }
-          )
-          .then(
-            function () {
-              return client.verifyEmail(code)
-            }
-          )
-          .then(
-            function () {
-              return client.emailStatus()
-            }
-          )
-          .then(
-            function (status) {
-              t.equal(status.verified, true, 'account is verified by confirming email')
-            }
-          )
-      }
-    )
-
-    test(
-      'Account verification links still work after session verification',
```

Comment:
The 2nd email in this case is sent via "recovery_email/resend_code", not a new "account/create". A 2nd call to "account/create" with the same email/password will generate a new account, the original link will be broken.